packages/contentful--create-contentful-app/src/includeFunction.ts

```diff
@@ -12,10 +12,11 @@ const addBuildCommand = getAddBuildCommandFn({
 });
 
 const VALID_FUNCTION_TEMPLATES_DIRS = [
-  'templates',
   'appevent-filter',
   'appevent-handler',
   'appevent-transformation',
+  'external-references',
+  'templates',
```

Comment:
Can't we just remove this?

Reply:
Probably we can, I am not sure what was the idea behind it...

Reply:
The idea behind is is that in the `apps` repo the example templates are in `examples/functions-templates`. A previous PR changed the `--functions` parameter to grab the template to be applied from `examples/functions-${cliArg}`. Because this was not on the radar when creating the `function-templates` folder the `--function external-references` will need to look in the `function-templates` folder. AFAIK the `templates` option is only there as it satisfies the `function-${cliArg}` format even though `--function templates` does not mean anything to CLI users.